towhee/dag/operator_repr.py

```diff
@@ -134,11 +134,7 @@ def from_dict(info: Dict) -> 'OperatorRepr':
         Returns:
             The operators we described in `file_or_src`
         """
-        if not BaseRepr.is_valid(info, {'name', 'init_args', 'function',
-                                        'inputs', 'outputs', 'iter_info'}):
-            raise ValueError(
-                'Invalid operator data.'
-            )
+        if not BaseRepr.is_valid(info, {'name', 'init_args', 'function', 'inputs', 'outputs', 'iter_info'}):
+            raise ValueError('Invalid operator info.')
 
-        return OperatorRepr(info['name'], info['function'], info['init_args'],
-                            info['inputs'], info['outputs'], info['iter_info'])
+        return OperatorRepr(info['name'], info['function'], info['init_args'], info['inputs'], info['outputs'], info['iter_info'])
```

Comment:
Does this pass our line length requirements? It's fairly long.

Reply:
fixed